<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbViewerWindow</class>
 <widget class="QMainWindow" name="dbViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Project : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="project_comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>db path : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dbPath_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Object ID Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="objectID_tableWidget">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>oID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AssetName</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AssetPath</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>mID</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_pushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Matte ID Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="matteID_tableWidget">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>mID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Color</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>MultiMatte</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>VrayMtl</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete2_pushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
